schema-next.yaml

```diff
@@ -63,6 +63,10 @@ versions:
               - jvm.buffer.usage
               - jvm.buffer.limit
               - jvm.buffer.count
+        - rename_attributes:
+          attribute_map:
+            net.host.name: server.domain # This is re-renamed from the 1.21.0 renaming to server.address
```

Comment:
I guess this is still a WIP? Also need to add the change to the affected metrics.

Reply:
It's not intentionally a WIP. Any way to check the syntax of this?